pkg/providers/aws/provider_redis.go

```diff
@@ -39,7 +39,7 @@ const (
 	defaultAtRestEncryption = true
 	defaultCacheNodeType    = "cache.t3.micro"
 	defaultDescription      = "A Redis replication group"
-	defaultEngineVersion    = "6.2"
+	defaultEngineVersion    = "7.1"
```

Comment:
another PR, please resolve

Reply:
please see previouse comments. Rebase tried, but complex, so it remains.
Thank you